app/content/tasks/event.py

```diff
@@ -57,7 +57,7 @@ def event_end_schedular(*args, **kwargs):
                 .add_paragraph(
                     "Undersøkelsen tar ca 1 minutt å svare på, og er til stor hjelp for fremtidige arrangementer. Takk på forhånd!"
                 )
-                .add_button("Åpne undersøkelsen", event.evaluate_link)
+                .add_evaluation_button(event)
```

Comment:
Tenker vi like gjerne kan bruke `.add_button` her. Evaluation button lages kun for akkurat denne eposten så ser ikke nytten i å lage en helt egen funksjon i MailCreator for denne.